src/components/FavoritedReportCard/FavoriteReportCard.js

```diff
@@ -2,18 +2,37 @@ import React from 'react';
 import './FavoriteReportCard.scss';
 import PropTypes from 'prop-types'
 
-const FavoriteReportCard = ({ name, studentTeacherRatio, instructionSalaryPercentOfTotal, perTeacherSalaryExpenses, enrollment, numberOfSchoolsInDistrict, studentGuidanceCounselorRatio, perStudentExpenditure }) => {
+const FavoriteReportCard = ({ name, studentTeacherRatio, perTeacherSalaryExpenses, enrollment, numberOfSchoolsInDistrict, studentGuidanceCounselorRatio, perStudentExpenditure }) => {
```

Comment:
why did we get rid of this instructional salary percent of total.  We agreed to display this yesterday.

Reply:
It was not rendering any information, so I assumed this was legacy code. My apologies!